src/Keyborg.ts

```diff
@@ -88,12 +93,27 @@ class KeyborgCore implements Disposable {
   private _win?: WindowWithKeyborg;
   private _isMouseUsed = false;
   private _dismissTimer: number | undefined;
+  private _triggerKeys?: Set<number>;
+  private _dismissKeys?: Set<number>;
 
-  constructor(win: WindowWithKeyborg) {
+  constructor(win: WindowWithKeyborg, props?: KeyborgProps) {
     this.id = "c" + ++_lastId;
     this._win = win;
     const doc = win.document;
 
+    if (props) {
+      const triggerKeys = props.triggerKeys;
```

Comment:
```suggestion
      const triggerKeys = props.triggerKeys ?? [];
```

That way you can get rid of the `if` clauses